pulumi/aws/anthos/__main__.py

```diff
@@ -141,7 +147,7 @@ def add_namespace(obj):
                                                       })
 
 # Configuration Values are stored in the configuration:
-#  ../config/Pulumi.STACKNAME.yaml
+#  ./config/Pulumi.STACKNAME.yaml
 config = pulumi.Config('anthos')
 demo_pwd = config.require_secret('demo_pwd')
 
```

Comment:
Let's delete the `demo_login` and `demo_pwd` bits.

Reply:
Done and updated in all other locations and configs.